packages/gridproxy_client/src/builders/gateways.ts

```diff
@@ -7,6 +7,13 @@ export enum NodeStatus {
   Standby = "standby",
 }
 
+export enum UnifiedNodeStatus {
+  Up = "up",
+  Down = "down",
+  Standby = "standby",
+  UpStandby = "up,standby",
```

Comment:
I'm not sure about the order, could be like what you did and could be like that.

```suggestion
  Up = "up",
  Standby = "standby",
  UpStandby = "up,standby",
  Down = "down",
```